src/SWP/Bundle/CoreBundle/Model/Route.php

```diff
@@ -14,222 +14,18 @@
 
 namespace SWP\Bundle\CoreBundle\Model;
 
-use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Collections\Collection;
 use SWP\Bundle\ContentBundle\Model\RouteInterface;
 use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
 use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
 use SWP\Bundle\ContentBundle\Model\Route as BaseRoute;
-use SWP\Component\Storage\Model\PersistableInterface;
 
 /**
  * Class Route.
  */
-class Route extends BaseRoute implements PersistableInterface, RouteInterface, TenantAwareInterface, ArticlesCountInterface
+class Route extends BaseRoute implements RouteInterface, TenantAwareInterface, ArticlesCountInterface
 {
     use TenantAwareTrait, ArticlesCountTrait;
 
-    /**
```

Comment:
why are all properties removed from this class?

Reply:
We have them already in extended class